docs/en/serverless/limitations.asciidoc

```diff
@@ -5,3 +5,10 @@
 // :keywords: serverless, observability
 
 Currently, the maximum ingestion rate for the Managed Intake Service (APM and OpenTelemetry ingest) is 11.5 MB/s of uncompressed data (roughly 1TB/d uncompressed equivalent). Ingestion at a higher rate may experience rate limiting or ingest failures.
+
+If you believe you are experiencing rate limiting or other ingest-based failures,
+please contact support@elastic.co for assistance.
+
+// TODO: Find out  the right way to tell users to reach out for support. Should we also mention the support portal...something like;
```

Comment:
@eedugon Do you have any thoughts/insight on what we should say to users when we want them to reach out to support? Should we give the support email? Send them to the support portal? Both?